HomeWorks/java/lesson3/Task3HW3Figure2.java

```diff
@@ -0,0 +1,11 @@
+package lesson3;
+
```

Comment:
where is the figure 3?
 static void triangle1(int n) {
        for (int i = 0; i < n; i++) {
            for (int j = i; j >= 0; j--) {
                System.out.print("*");
            }
            System.out.println();
        }
        for (int a = n - 1; a > 0; a--) {
            for (int b = a; b > 0; b--) {
                System.out.print("*");
            }
            System.out.println();
        }
    }